packages/@ourworldindata/grapher/src/chart/ChartManager.ts

```diff
@@ -33,6 +33,7 @@ export interface ChartManager {
     transformedTable?: OwidTable
 
     variant?: GrapherVariant
+    isDisplayedAlongsideComplementaryTable?: boolean
```

Comment:
Just to double-check: we can't use the existing `showLegend`, because the new field affects more than just the legend?